src/components/syncs/datasets/SyncDataSet.js

```diff
@@ -152,24 +161,32 @@ const SyncDataSet = (props) => {
                   after: 5,
                 }}
                 onPeriodChange={(newPeriod) => {
-                  props.history.push("/sync/datasets/" + newPeriod);
+                  const newUrl = window.location.href.replace(
+                    "/sync/datasets/" + period,
+                    "/sync/datasets/" + newPeriod,
+                  );
+                  window.history.pushState({}, "", newUrl);
```

Comment:
may be should find a "better way" then reloading. 
perhaps the react router history replace ?

Reply:
I've tested it and I seem to lose the filters when using router history's replace, but I may be missing something